compute_endpoint/globus_compute_endpoint/engines/globus_compute.py

```diff
@@ -32,12 +32,14 @@ def __init__(
         *args,
         label: str = "GlobusComputeEngine",
         max_retries_on_system_failure: int = 0,
-        strategy: t.Optional[SimpleStrategy] = SimpleStrategy(),
         executor: t.Optional[HighThroughputExecutor] = None,
         container_type: t.Literal[VALID_CONTAINER_TYPES] = None,  # type: ignore
         container_uri: t.Optional[str] = None,
         container_cmd_options: t.Optional[str] = None,
         encrypted: bool = True,
+        max_idletime: int = 120,
+        strategy: t.Optional[str] = "simple",
+        job_status_kwargs: t.Optional[t.Dict] = None,
```

Comment:
As discussed out-of-band, we'll need to update the EngineModel.strategy config model field to accept either `str` or `StrategyModel` and add a `EngineModel.job_status_kwargs` field.

Reply:
@rjmello, I talked to @joshbryan-globus yesterday and the plan is to add configurability from the config.yaml file as a follow-up PR. Right now this PR is functional, and these options really wouldn't need to be touched by the user.

Reply:
Works for me.